test/Elastic.Apm.AspNetCore.Tests/AspNetCoreMiddlewareTests.cs

```diff
@@ -74,6 +74,13 @@ public async Task HomeSimplePageTransactionTest()
 
 			//test transaction.context.response
 			transaction.Context.Response.StatusCode.Should().Be(200);
+			if (_agent.ConfigurationReader.CaptureHeaders)
+			{
+				transaction.Context.Response.Headers.Should().NotBeNull();
+				transaction.Context.Response.Headers.Should().NotBeEmpty();
+			}
+			else

```

Comment:
Do we expect this `else` block to be executed? If I understand it correctly we don't parametrize this test, and by default we always capture headers, so we only need the two asserts from the `if` block.